src/main/java/org/candlepin/model/AnonymousCloudConsumer.java

```diff
@@ -50,8 +54,8 @@ public class AnonymousCloudConsumer extends AbstractHibernateObject<AnonymousClo
     public static final int CLOUD_INSTANCE_ID_MAX_LENGTH = 85;
     /** Max length for a value in the cloud provider short name field */
     public static final int CLOUD_PROVIDER_SHORT_NAME_MAX_LENGTH = 15;
-    /** Max length for a value in the product ID field */
-    public static final int PRODUCT_ID_MAX_LENGTH = 32;
+    /** Max length for a value in the product IDs field */
+    public static final int PRODUCT_ID_MAX_LENGTH = 200;
```

Comment:
So this doesn't seem to be used any more. Especially with the db column having changed to TEXT, it can be removed right? Alternatively, we can keep the column as VARCHAR with a large enough value.

Reply:
Good catch! Yeah, this should be removed. I missed this when I switched the column from VARCHAR to TEXT. Removed